unrpyc.py

```diff
@@ -262,43 +262,34 @@ def main():
             print("\n".join(e.args))
             return
 
-    # Expand wildcards
-    def glob_or_complain(s):
-        retval = glob.glob(s)
-        if not retval:
-            print("File not found: " + s)
-        return retval
-    filesAndDirs = [glob_or_complain(i) for i in args.file]
-    # Concatenate lists
-    filesAndDirs = list(itertools.chain(*filesAndDirs))
-
-    # Recursively add .rpyc files from any directories passed
-    files = []
-    for i in filesAndDirs:
-        if path.isdir(i):
-            for dirpath, dirnames, filenames in walk(i):
-                files.extend(path.join(dirpath, j) for j in filenames if len(j) >= 5 and j.endswith(('.rpyc', '.rpymc')))
-        else:
-            files.append(i)
-
-    # Check if we actually have files. Don't worry about
-    # no parameters passed, since ArgumentParser catches that
-    if len(files) == 0:
-        print("No script files to decompile.")
+    def rpyc_check(inp):
+        return bool(inp.suffix in ('.rpyc', '.rpymc') and inp.is_file())
+
+    files = list()
+    for item in args.file:
```

Comment:
Right now this seems to not actually process globs given as input commands correctly.